src/nuclio/functions/version/version-code/version-code.component.js

```diff
@@ -550,26 +554,14 @@
                 });
         }
 
-        /**
-         * Broadcast callback from angular-ui-layout.
-         * Sets new width of test pane.
-         * @param event
-         * @param beforeContainerResize
-         * @param afterContainerResize
-         */
-        function onLayoutResize(event, beforeContainerResize, afterContainerResize) {
-            testPaneWidth = afterContainerResize.size;
-
-            resizeScrollBar(100);
-        }
-
         /**
          * Resize scrollbar container.
          * Layout directive (splitter) makes changes to width of scrollbar container. But scrollbar doesn't handle
          * those changes in correct way. So we have to set width manually
+         * @param {Event} e - native broadcast event object
          * @param {number} timeout - function invocation delay
```

Comment:
If it now has a default value you should specify it here in JSDoc:
```suggestion
         * @param {number} [timeout=200] - function invocation delay in milliseconds
```